js/player.js

```diff
@@ -1,55 +0,0 @@
-function playerTank() {
-  player.push({
-    x: 180,
-    y: 500,
-    w: 65,
-    h: 65,
-  });
-}
-
-function drawPlayer(player) {
-  image(imgPlayer, player.x, player.y, player.w, player.h);
-}
-
-function movement(player) {
-  if (player.x > 420) {
-    player.x = -50;
-  }
-  if (player.x < -50) {
-    player.x = 400;
```

Comment:
hardcoded magical numbers -> make them dependent on width and height of the canvas instead :) 
then it would work on any size of screen easily